src/hotspot/cpu/s390/vm_version_s390.hpp

```diff
@@ -121,6 +121,7 @@ class VM_Version: public Abstract_VM_Version {
 // --- FeatureBitString Bits 193..200 (DW[3]) ---
 // ----------------------------------------------
 #define  BEAREnhFacilityMask            0x4000000000000000UL  // z16, BEAR-enhancement facility, Bit: 193
+#define  ConcurrentFunFacilityMask      0x0040000000000000UL  // z17, Concurrent-functions facility, Bit: 201
```

Comment:
I think above comment in Line 121 is incorrect about bits covered in DW[3] and seems like comment about DW[2] is also incorrect. Can you update those as well. 

```
DW[2] should cover bits: 128–191
DW[3] should cover bits: 192–255
```